src/spot_moveit_config/config/spot.srdf

```diff
@@ -26,132 +24,69 @@
         <joint name="rear_right_hip_y"/>
         <joint name="rear_right_knee"/>
     </group>
-    <group name="gripper">
-        <joint name="arm_gripper"/>
+    <group name="arm">
+        <chain base_link="body" tip_link="arm_link_wr1"/>
+    </group>
+    <group name="end_effector">
+        <joint name="arm_f1x"/>
     </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="default_standing" group="spot_body">
+    <group_state name="standing" group="body">
         <joint name="front_left_hip_x" value="0"/>
-        <joint name="front_left_hip_y" value="0.872665"/>
-        <joint name="front_left_knee" value="-1.44862"/>
-
+        <joint name="front_left_hip_y" value="0"/>
+        <joint name="front_left_knee" value="-1.52385"/>
         <joint name="front_right_hip_x" value="0"/>
-        <joint name="front_right_hip_y" value="0.872665"/>
-        <joint name="front_right_knee" value="-1.44862"/>
-
+        <joint name="front_right_hip_y" value="0"/>
+        <joint name="front_right_knee" value="-1.52023"/>
         <joint name="rear_left_hip_x" value="0"/>
-        <joint name="rear_left_hip_y" value="0.872665"/>
-        <joint name="rear_left_knee" value="-1.44862"/>
-
+        <joint name="rear_left_hip_y" value="0"/>
+        <joint name="rear_left_knee" value="-1.53003"/>
         <joint name="rear_right_hip_x" value="0"/>
-        <joint name="rear_right_hip_y" value="0.872665"/>
-        <joint name="rear_right_knee" value="-1.44862"/>
+        <joint name="rear_right_hip_y" value="0"/>
+        <joint name="rear_right_knee" value="-1.52531"/>
     </group_state>
-    <group_state name="arm_stowed" group="spot_arm">
-        <joint name="arm_joint1" value="1.5878"/>
-        <joint name="arm_joint2" value="0"/>
-        <joint name="arm_joint3" value="2.967"/>
-        <joint name="arm_joint4" value="0.0468"/>
-        <joint name="arm_joint5" value="0"/>
-        <joint name="arm_joint6" value="0"/>
+    <group_state name="arm_stowed" group="arm">
+        <joint name="arm_el0" value="3.1415"/>
+        <joint name="arm_el1" value="0"/>
+        <joint name="arm_sh0" value="0"/>
+        <joint name="arm_sh1" value="-3.1415"/>
+        <joint name="arm_wr0" value="0"/>
+        <joint name="arm_wr1" value="0"/>
     </group_state>
```

Comment:
Does the `arm_stowed` state look correct?

Reply:
Yes. The arm is folded as expected yet does not collide with the robot's shoulder or itself.